src/events/handler.js

```diff
@@ -15,7 +15,10 @@ import mouseDown from "./mouseDown";
 import mouseMove from "./mouseMove";
 import mouseUp from "./mouseUp";
 
-const eventHandler = (state, dispatch, allowEdit) => (event) => {
+const eventHandler = (state, dispatch, allowEdit, eventCallback) => (event) => {
+  if (eventCallback !== undefined) {
```

Comment:
https://github.com/nextml-code/lodestar/blob/master/src/isDefined.js#L20

Reply:
If we wanna add lodestar we should probably refactor most of everything else too :)

Reply:
ah, image-annotation-canvas is perhaps pre lodestar
yeah agree, PRs should be one consern at a time